aten/src/ATen/native/cuda/Loss.cu

```diff
@@ -71,24 +71,26 @@ Tensor& binary_cross_entropy_out_cuda(Tensor& loss, const Tensor& input, const T
   iter.add_input(at::squeeze(input));
   iter.add_input(at::squeeze(target));
   iter.build();
-  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.common_dtype(), "binary_cross_entropy_out_cuda", [&]() {
-    gpu_kernel(iter,
-      [] GPU_LAMBDA (scalar_t input_val, scalar_t target_val) -> scalar_t {
-        const scalar_t zero = 0;
-        const scalar_t one = 1;
-        const scalar_t neg_100 = -100;
+  AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::BFloat16, iter.common_dtype(), "binary_cross_entropy_out_cuda", [&]() {
+    AT_SKIP_BFLOAT16_IF_NOT_ROCM(scalar_t, "binary_cross_entropy_out_cuda", [&] {
+      gpu_kernel(iter,
+        [] GPU_LAMBDA (scalar_t input_val, scalar_t target_val) -> scalar_t {
+          const scalar_t zero = 0;
+          const scalar_t one = 1;
+          const scalar_t neg_100 = -100;
 
-        CUDA_KERNEL_ASSERT(input_val >= zero && input_val <= one);
+          CUDA_KERNEL_ASSERT(input_val >= zero && input_val <= one);
```

Comment:
This and below are only superfluous white space changes, or? If yes, please remove.

Reply:
No, these are not superfluous white space changes, just additional indentation since it is now wrapped inside `AT_SKIP_BFLOAT16_IF_NOT_ROCM` macro.